charts/datadog/ci/kubeconform-values.yaml

```diff
@@ -36,7 +36,7 @@ clusterAgent:
   enabled: true
   createPodDisruptionBudget: true
   nodeSelector:
-    kubernetes.io/os: linux
+    disktype: ssd
```

Comment:
does this change is needed?

Reply:
See comment above and #1312 - short answer is yes if we want to have `-strict` and the intent is to allow multiple (different) values to be passed in to `nodeSelector`.

Reply:
thanks for the explanation.